locations/spiders/omv.py

```diff
@@ -0,0 +1,187 @@
+        iconf_ts = re.search(r"var IConfTs = '([^']+)'", javascript_code).group(1)
+
+        for brand, brand_data in BRANDS_AND_COUNTRIES.items():
+            for country in brand_data["countries"]:
+                yield scrapy.FormRequest(
+                    url=self.api_url,
+                    formdata={
+                        "CTRISO": country,
+                        "BRAND": brand,
+                        "VEHICLE": "CAR",
+                        "MODE": "NEXTDOOR",
+                        "ANZ": "1000",
+                        "HASH": iconf_hash,
+                        "TS": iconf_ts,
+                    },
+                    callback=self.parse_pois,
+                    meta={
+                        "country": country,
+                        "brand": brand_data["brand"],
+                        "brand_wikidata": brand_data["brand_wikidata"],
+                        "HASH": iconf_hash,
+                        "TS": iconf_ts,
+                    },
+                )
+
+    def parse_pois(self, response):
+        for poi in response.json():
+            yield scrapy.FormRequest(
+                url=self.details_url,
+                formdata={
+                    "ID": poi["sid"],
+                    "HASH": response.meta["HASH"],
+                    "TS": response.meta["TS"],
+                },
+                callback=self.parse_poi,
+                meta=response.meta,
+            )
+
+    def parse_poi(self, response):
+        data = response.json()
+        details = data.get("siteDetails", {})
+        item = DictParser.parse(details)
+        item["ref"] = details.get("sid")
+        item["lat"] = details.get("y_coordinates")
+        item["lon"] = details.get("x_coordinates")
+        item["street_address"] = details.get("address_l")
+        item["city"] = details.get("town_l")
+        item["phone"] = details.get("telnr") if details.get("telnr") != "NO TELEPHONE" else None
+        item["brand"] = response.meta["brand"]
+        item["brand_wikidata"] = response.meta["brand_wikidata"]
+        item["country"] = response.meta["country"]
+        self.parse_hours(item, details)
+        self.parse_attribute(item, data, "siteFeatures", SITE_FEATURES_MAP)
+        self.parse_attribute(item, data, "paymentDetails", PAYMENT_METHODS_MAP)
+        apply_category(Categories.FUEL_STATION, item)
+        # TODO: fuel types are provided as images, parse them somehow. OCR?
+        yield item
+
+    def parse_hours(self, item, details):
+        """
+        Example opening hours string:
+            "dayOfWeek=1,closed=FALSE,from=05:00,to=22:00#dayOfWeek=2,closed=FALSE,from=05:00,to=22:00"
+        """
+        oh = OpeningHours()
+        hours = details.get("opening_hours")
+        try:
+            if hours:
+                for day in hours.split("#"):
+                    dayOfWeek, closed, fromTime, toTime = day.split(",")
```

Comment:
I would have parsed this into a dict, instead of trusting the order will always be the same. Something like:
```python
    for raw_rule in hours.split("#"):
        rule = {}
        for var in raw_rule.split(","):
            key, value = var.split("=", 1)
            rule[key] = value
```